localization/en-us.lua

```diff
@@ -411,16 +434,17 @@ return {
             b_Bakery_double_tags = "Double Tags",
             -- KEEP_LITE
             k_bakerycharm = "Charm",
-            k_BakeryCharmInfo = { "Only one charm may be equipped at a time,",
-                "purchasing a new charm replaces an old one." },
+            k_BakeryCharmInfo = { "Only one Charm may be equipped at a time.",
+                "Purchasing a new Charm replaces an old one." },
             b_Bakery_equip = "EQUIP",
             b_Bakery_ante = "(Ante)"
         },
         -- END_KEEP_LITE
         v_dictionary = {
             b_Bakery_deposit = "DEPOSIT $#1#",
             b_Bakery_ante_times = "(Ante*#1#)",
-            v_Bakery_artist = "Artist: #1#"
+            v_Bakery_artist = "Artist: #1#",
+            v_Bakery_coder = "Coder: #1#"
```

Comment:
I'm not entirely fond of the term "`Coder`" for this. Perhaps this should become `"Art: #1#"` and `"Code: #1#"`.